sui_types/src/messages.rs

```diff
@@ -1081,7 +1081,7 @@ impl CertifiedTransaction {
             is_checked: false,
             data: transaction.data,
             tx_signature: transaction.tx_signature,
-            auth_sign_info: QuorumSignInfo {
+            auth_sign_info: AuthorityQuorumSignInfo {
```

Comment:
Thanks, the readability problem is still remaining (cause while you read this you expect to see an `AuthoritySignInfo`),  what about renaming `auth_sign_info` to `quorum_sign_info` and keep your previous `QuorumSignInfo`?

Reply:
auth_sign_info can be one of EmptySignInfo, AuthoritySignInfo and AuthorityQuorumSignInfo.
So I don't think we want to rename it to quorum_sign_info.

Reply:
Hm.. ok, let's leave it unchanged for now, we might consider renaming it to the more generic `sign_info` later, but don't change yet, `auth_sign_info` works for now.

Reply:
The same structure also contains transaction signature (tx_signature). So this is named auth_sign_info to emphasize that the signature is from authority side instead of from transaction signer.
But yeah open to better names!